src/routes/api/role.routes.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import RoleController from '../../controllers/role.controller';
+import Authentication from '../../middlewares/authentication';
+import { roleValidation } from '../../middlewares/roleValidations';
+
+const { assignRole } = RoleController;
+const {
+    isUserLoggedInAndVerified
+} = Authentication;
+
+const roleRoutes = express.Router();
+
+roleRoutes.patch('/assignrole', isUserLoggedInAndVerified, roleValidation, assignRole);
```

Comment:
Although doing so doesn't matter but I think a good way of writing routes should be like this `assign-role` not `assignrole`